app/src/main/res/drawable/ic_settings.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
what is the purpose of this file?

Reply:
I think it's the settings icon for log out that connor converted from svg to vector drawable

Reply:
Yes ^ the one in AccountInfoFragment